raiden-dapp/tests/unit/views/transfer-steps.spec.ts

```diff
@@ -240,4 +247,47 @@ describe('TransferSteps.vue', () => {
     expect(transferDone).toBeCalledTimes(0);
     expect(wrapper.vm.$data.error).toEqual('failure');
   });
+
+  test('skip to transfer if a direct route is available', async () => {
```

Comment:
`'Should...'`

Reply:
When it comes to jasmine's `it` `Should` sounds like the natural next step. Is that a common convention for jest test naming too?

Reply:
That's what we have always used in the past, I guess it has not so much to do with jest or jasmine but it's one of many naming conventions. Maybe we can decide on one in the test refactor ticket and just leave them here as they are? Here are more "patterns": https://dzone.com/articles/7-popular-unit-test-naming

PS: `test('Should fail when x y z')` also sounds pretty natural :)

Reply:
@andrevmatos it would be nice to have a consistent naming strategy in the tests accross the SDK in general it would be nice to also have your input here.

Reply:
On the SDK, `test` doesn't use `should`, I usually just name the test in a descriptive way, like `test('fail when amount is negative', () => ...`

Reply:
I don't mind doing this way either and since I am touching all the tests anyway it would not be much @nephix are you ok with changing the tests in the dapp to follow the same scheme?

Reply:
Sure sounds good!